frontend/src/components/VCollectionHeader/meta/VCollectionHeader.stories.mdx

```diff
@@ -5,10 +5,10 @@ import {
   Meta,
   Story,
 } from "@storybook/addon-docs"
+import { useMediaStore } from "~/stores/media"
 import { useProviderStore } from "~/stores/provider"
 
 import VCollectionHeader from "~/components/VCollectionHeader/VCollectionHeader.vue"
-import { useMediaStore } from "@/stores/media"
```

Comment:
Should have a lint for this. Thanks for catching!

Reply:
We probably don't have it set up for mdx files. I think this story should also have been deleted, though.